chore: switch to new Aspect Workflows OSS staging deployment (#258)
aspect-build/rules_jest: .aspect/workflows/README.md

    commands:
      - "rosetta steps | buildkite-agent pipeline upload"
    agents:
      queue: aspect-nano-graviton
```

> [!IMPORTANT]

> > The Setup Aspect Workflows step above is configured run on the `aspect-nano-graviton` queue which is serviced by the a runner group made up of light weight and inexpensive `t4g.nano` AWS instances.

Reviewer: ```suggestion
> The Setup Aspect Workflows step above is configured to run on the `aspect-nano-graviton` queue that is serviced by a runner group made up of lightweight and inexpensive `t4g.nano` AWS instances.
```
